### ISSUE TITLE
Unable to import module 'wsgi': No module named 'werkzeug'

### ISSUE THREAD

**Issue by babyhuey** [logandk/serverless-wsgi#80]
Suddenly getting this error again on v1.6.1

Rolling back to 1.6.0 fixes the issue

**Comment by logandk** [comment]
Are you using `serverless-python-requirements`?

**Comment by logandk** [comment]
And you have `werkzeug` in your `requirements.txt`? There's no change in `1.6.0` -> `1.6.1` that should cause and import error for werkzeug. I'd recommend unzipping the zip bundle that is uploaded to lambda and verify that werkzeug is present.

**Comment by babyhuey** [comment]
I also just realized that I said 1.6.0 originally, when I rolled back to 1.5.3. 

So I'm guessing that it is from this change? 
Detect presence of serverless-python-requirements and disable packRequirements automatically

If so, documentation should probably be updated saying to add werkzeug to requirements if using python-serverless-requirements plugin

**Comment by logandk** [comment]
Right, then you probably didn't have `packRequirements` set to `false` before, meaning that both `serverless-wsgi` and `serverless-python-requirements` were installing your pip packages (and only `serverless-wsgi` always makes sure to include `werkzeug`).

I agree that this should be added to the documentation, will keep the issue open until this is completed.

**Comment by fluki-mx** [comment]
I found this in stackoverflow and it works I downgrade Flask to 1.1.4

> Flask, Werkzeug and other pallets projects just had a major update, dropping python2 support and deleting _compat module. And AWS has't resolve the capability issue yet.
> The simplest fix will be downgrading Flask, Werkzeug, etc. to the previous major version.

[Credits to Allan Chain](https://stackoverflow.com/a/67554060/4010240)

**Comment by mohammad-aqajani** [comment]
my solution: 
https://github.com/serverless/serverless-python-requirements/issues/469#issuecomment-1181742036
